packages/typedoc-plugin-appium/lib/converter/comment/types.ts

```diff
@@ -0,0 +1,31 @@
+import {Comment, Reflection} from 'typedoc';
+import {CommentSource, KnownMethods} from '../types';
+
+/**
+ * A function which queries some object for a {@linkcode Comment}, and if found,
+ * its {@linkcode Comment.summary summary}.
```

Comment:
should this be `{@linkcode Comment.summary} summary`?

Reply:
it does the same thing:

![image](https://user-images.githubusercontent.com/924465/219764319-995669e0-5a8c-4ecd-850d-f86f1db5d9bb.png)